__tests__/bookmarks.ts

```diff
@@ -544,6 +562,153 @@ describe('query bookmarks', () => {
     expect(res.data).toMatchSnapshot();
   });
 
+  describe('plus user', () => {
+    beforeEach(async () => {
+      loggedUser = '5';
+      isPlus = true;
+      await con.getRepository(Settings).save({ userId: loggedUser });
```

Comment:
Basically when testing plus users, users need to have a settings. Otherwise, the query might return values that are not expected :hide_the_pain: - spent some good hours on this.